tree/tree/src/TTree.cxx

```diff
@@ -9578,6 +9615,8 @@ void TTree::UseCurrentStyle()
 Int_t TTree::Write(const char *name, Int_t option, Int_t bufsize) const
 {
    FlushBasketsImpl();
+   if (R__unlikely(option & kOnlyPrepStep))
```

Comment:
That isn't called in a hot loop; no need for the extra characters and complexity of `R__unlikely`. Or what am I missing?

Reply:
That we still know that it is unlikely :)